include/graphblas/nonblocking/blas2.hpp

```diff
@@ -1518,11 +1519,12 @@ namespace grb {
 		internal::le.execution();
 
 		// second, delegate to the reference backend
-		return eWiseLambda< ActiveDistribution, Func, DataType, RIT, CIT, NIT >(
+		return eWiseLambda< descr, ActiveDistribution, Func, DataType, RIT, CIT, NIT >(
 			f, internal::getRefMatrix( A ), s, P );
 	}
 
 	template<
+		Descriptor descr = descriptors::no_operation,
```

Comment:
This descriptor is not passed to the subsequent call to eWuseLambda (line 1551 below)